SampleCode/SwiftSampleCode/DronecodeSDKSwiftDemo/Controllers/MapViewController.swift

```diff
@@ -161,6 +215,19 @@ class MapViewController: UIViewController {
         
     }
     
+    // MARK: - Location manager
+    func locationManager(_ manager: CLLocationManager, didUpdateLocations locations: [CLLocation]) {
+        defer { currentLocation = locations.last }
+        
+        /*if currentLocation == nil {
```

Comment:
Doesn't that work? Should we remove the commented part then?

Reply:
indeed we could remove. It works but the commented code zoom on the user location when the app get it, so I prefer to let the zoom on the drone

Reply:
Right. So let's remove the commented code, otherwise it adds unnecessary noise.